src/gl/special_case.cpp

```diff
@@ -55,7 +55,16 @@ uint32_t gl_special_case(uint32_t primitivetype, uint32_t vertextype, struct nve
 	if(ff8) current_state.texture_filter = enable_bilinear && vertextype != TLVERTEX && current_state.texture_set;
 	else if (enable_bilinear && (vertextype != TLVERTEX || mode == MODE_MENU || (current_state.texture_set && VREF(texture_set, ogl.gl_set->force_filter)))) current_state.texture_filter = true;
 
-	// some modpath textures have z-sort forced on
+  // since we are no longer disabling all filtering on interal textures, we need to shut it off for the ones that filtering breaks.
+  
```

Comment:
Is this new line intended? If not, please remove it.

Reply:
Figured out what you meant. corrected.